doc/doxygen/headers/coding_conventions.h

```diff
@@ -119,7 +119,33 @@ executable.
   then functions.
   <br>
   Exceptions shall be declared at the end of the public section
-  before the non-public sections start.</li>
+  before the non-public sections start.
+  <br>
+  We do not use the C++11-style class member initialization for member variables
+  that are neither <code>static const</code> nor <code>static constexpr</code>;
+  i.e., instead of
```

Comment:
do we provide rationales for the other rules, or do we just say that that's what it is? if the former, feel free to add a sentence explaining why.

Reply:
We just enumerate the rules here: nothing else under 'style issues' gives an explanation.